lib/ueberauth/strategy/strava.ex

```diff
@@ -3,9 +3,7 @@ defmodule Ueberauth.Strategy.Strava do
   Strava Strategy for Überauth.
   """
 
-  use Ueberauth.Strategy, default_scope: "public",
-                          uid_field: :id,
-                          oauth2_module: Ueberauth.Strategy.Strava.OAuth
+  use Ueberauth.Strategy, default_scope: "view_private"
```

Comment:
I'd prefer the default_scope to be `public` still. I'm curious what your thought process was in making the default private?